### PR TITLE
chore: Allow Webpack v3.x as a peerDependency

### PR DESCRIPTION
**Pull request by fb55**

### CODE REVIEW

**Comment by danez** [comment]
@TrySound @d3viant0ne Can we merge this and release a new patch version? I know it is part of #38, but it seems that #38 probably needs some more time and this change is really simple :)

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     }
   },
   "peerDependencies": {
-    "webpack": ">=0.9 <2 || ^2.1.0-beta || ^2.2.0"
+    "webpack": ">=0.9 <2 || ^2.1.0-beta || >=2.2.0 <4"
```
Comment (michael-ciniawsky) [inline package.json]: `^1.0.0 || ^2.0.0 || ^3.0.0`

Comment (fb55) [inline package.json]: That would include `2.0.x` again and disallow `0.9`, which I'd consider a breaking change (semver major).

Comment (michael-ciniawsky) [inline package.json]: There was never a `2.0.x` the first release without `-beta` was `v2.2.0` to my knowledge. Yep for the `>= 0.9` if _really_ needed just  `>= 0.9 || ^1.0.0 || ^2.0.0 || ^3.0.0` then imho

Comment (michael-ciniawsky) [inline package.json]: or `>= 0.9 < 4` or even `< 4` ?

**Comment by mikegreiling** [comment]
> @TrySound @d3viant0ne Can we merge this and release a new patch version? I know it is part of #81, but it seems that #81 probably needs some more time and this change is really simple :)

yes, please!  adding this in a patch release means we won't have to bug libs that depend on `worker-loader` to update their dependency version.  (for us it's https://github.com/mozilla/pdfjs-dist)

File: package.json (hunk 1 above)
Comment (d3viant0ne) [inline package.json]: @danez - Yes, we can land a quick patch. 

@fb55 - Set the peerDependencies to  `"webpack": ">=0.9 || ^2.0.0 || ^3.0.0"`

**Comment by renchap** [comment]
@d3viant0ne any plans to publish this version on NPM?

**Comment by renchap** [comment]
@d3viant0ne Friendly bump as you took care of https://github.com/webpack-contrib/coffee-loader/issues/39

You merged Webpack 3 support but did not release on NPM, preventing users to upgrade to Webpack 3 (corresponding issue: https://github.com/webpack-contrib/worker-loader/issues/75)

Thanks! 🍰

**Comment by ianks** [comment]
yes @d3viant0ne please create a release :)

**Comment by davidagee** [comment]
any update on releasing this? We're currently blocked on shrinkwrapping a release due to npm ...interesting... behavior wrt peerDependencies and shrinkwrap, and can't update npm until after this release. Bit of a catch-22.

**Pull request merged**